installation_guides/ubuntu.md

```diff
@@ -0,0 +1,75 @@
+## Set up guide for Ubuntu
+
+This document is intended to provide you with the details for how to install and
+configure conda, python, pip, pytest, pyspark and dask.
+Note: these instructions were tested on Ubuntu 18.04.3.
+
+
+### Installing Conda
+
+Conda is an open-source package management system and environment management system that runs on Windows, macOS, and Linux. We will use Conda to create a virtual environment with python3.5, PySpark, Dask and PyTest.
+
+1. Download Miniconda for Linux by executing the following command:
+
+```
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(uname -i).sh
+```
+2. Grant execution rights to the installer with the command `chmod +x Miniconda3-latest-Linux-$(uname -i).sh`.
+3. Execute the installer by executing the command `./Miniconda3-latest-Linux-$(uname -i).sh`. Press ENTER, then if you accept the license agreement type `yes` and press ENTER. Enter the path where you want to install anaconda on your computer (for example `~/.condainstallation`). Finally, type `yes` to let the installer initialize conda.
+4. Reinitialize your shell by executing the command `source ~/.bashrc`.
+5. To prevent conda from activating the default environment whenever you open a shell, use the following command: `conda config --set auto_activate_base false`.
+6. Execute the command `conda update -y -n base -c defaults conda` to update conda to its latest version.
+7. Congrats! You have successfully installed conda.
+
+### Creating a virtual environment
+
+Conda can be used to create environments. A conda environment is a directory that contains a specific collection of conda packages. They make it possible to have different versions of python and python packages installed on the same system in different environments.
+
+1. Create a virtual environment with python 3.5 by executing the command: `conda create -n bigdata-lab python=3.5 -y`
+2. Activate your virtualenv by executing the command: `conda activate bigdata-lab`
+
+
+### Installing PySpark inside your virtual environment
+
+PySpark is the Python API in Apache Spark. It is required to complete the course assignments.
+In this section, we will install Apache Spark into the virtual environment that we
+have just created.
+
+1. With your venv activated, we will execute the command `conda install -y pyspark`.
+   You have just installed PySpark, however, Apache Spark needs the Java Platform to run.
```

Comment:
for some reason installing PySpark like this will install it for python 3.7.4. Is this the case for you? (I double checked and i had installed python3.5 in my conda environment and python3 doesn't exist outside of it)

Reply:
my bad! was in the base environment.